tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3520 @@
+              * `12` - denotes that the chain state transition is invalid.
+              * `13` - denotes that the referenced input is created after transaction issuing time.
+              * `14` - denotes that the mana amount is invalid.
+              * `15` - denotes that the Block Issuance Credits Input is invalid.
+              * `16` - denotes that Reward Context Input is invalid.
+              * `17` - denotes that Commitment Context Input is invalid.
+              * `18` - denotes that Staking Feature is not provided in account output when claiming rewards.
+              * `19` - denotes that fail to claim staking reward.
+              * `20` - denotes that fail to claim delegation reward.
+              * `21` - denotes that the burning of native tokens was not allowed in the transaction capabilities.
+              * `22` - denotes that the burning of mana was not allowed in the transaction capabilities.
+              * `23` - denotes that the destruction of accounts was not allowed in the transaction capabilities.
+              * `24` - denotes that the destruction of anchors was not allowed in the transaction capabilities.
+              * `25` - denotes that the destruction of foundries was not allowed in the transaction capabilities.
+              * `26` - denotes that the destruction of nfts was not allowed in the transaction capabilities.
+              * `255` - denotes that the semantic validation failed.
+      required:
+        - blockId
+        - blockState
+
+    BlockResponse:
+      description: Returns a given block.
+      properties:
+        allOf:
+          $ref: '#/components/schemas/Block'
+
+    OutputResponse:
+      description: Returns an output.
+      properties:
+        output:
+            anyOf:
+              - $ref: '#/components/schemas/BasicOutput'
+              - $ref: '#/components/schemas/AccountOutput'
+              - $ref: '#/components/schemas/AnchorOutput'
+              - $ref: '#/components/schemas/FoundryOutput'
+              - $ref: '#/components/schemas/NFTOutput'
+              - $ref: '#/components/schemas/DelegationOutput'
+        outputIdProof:
+          allOf:
+              - $ref: '#/components/schemas/OutputIdProof'
+      required:
+        - output
+        - outputIdProof
+          
+    OutputIdProof:
+      description: The proof of the output identifier.
+      properties:
+        slot:
+          type: integer
+          description: The slot index of the output.
+        outputIndex:
+          type: integer
+          description: The index of the output.
+        transactionCommitment:
+          type: string
+          description: The commitment of the transaction that created the output. Hex-encoded with 0x prefix.
+        outputCommitmentProof:
+          type: string
+          description: The proof of the output commitment. Hex-encoded with 0x prefix.
```

Comment:
Is this correct? Reading https://github.com/iotaledger/tips/blob/tip45/tips/TIP-0045/tip-0045.md#output-id-proof it looks like this should be an object